web/modules/custom/joinup_comment/config/install/field.field.message.comment_deletion.field_page.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.message.field_page
+    - message.template.comment_deletion
+id: message.comment_deletion.field_page
```

Comment:
- joinup_notification makes some assumptions on the structure of things
(comments have to lookup their embedding entity)
With some effort we can probably try to make this fit the comment use case as well...
- reference fields can only have one target type.
- This would introduce a dependency on joinup_notifications